frontend/src/v5/ui/components/legal/LegalLayout/legalAppBar/legalAppBar.component.tsx

```diff
@@ -35,9 +35,9 @@ export const LegalAppBar = ({ activePage }: ILegalAppBar): JSX.Element => (
 				<LogoIcon />
 			</Link>
 			<NavLinks>
-				<NavLink to="/v5/privacy" selected={activePage === 'privacy'}>
+				<PrivacyLink href={PRIVACY_ROUTE} target="_blank">
```

Comment:
Can we fix the formatting here too:
staging:
![image](https://github.com/3drepo/3drepo.io/assets/11945337/c889f2fc-3683-4183-b741-622f67b0263f)

this branch:
![image](https://github.com/3drepo/3drepo.io/assets/11945337/4a0686c7-2c50-4d73-94ab-c2379fcf33bc)